ports/esp32/main.c

```diff
@@ -257,3 +257,15 @@ void *esp_native_code_commit(void *buf, size_t len, void *reloc) {
     memcpy(p, buf, len);
     return p;
 }
+
+
+// Workaround for https://github.com/espressif/esp-insights/issues/38
+extern int  __real_esp_efuse_rtc_calib_get_ver(void);
+int __wrap_esp_efuse_rtc_calib_get_ver(void)
+{
+    #if MICROPY_PI_QEMU
```

Comment:
I don't know if this is the correct name for such a config option.

Is there a macro option that the IDF defines when building for qemu that can be used here instead?

Reply:
The idf build has nothing special when targeting qemu. Enabling openeth is the only way to tell it's a qemu build

Reply:
switched it to CONFIG_ETH_USE_OPENETH instead and removed custom config option